consensus/proto_array/src/proto_array.rs

```diff
@@ -334,54 +334,54 @@ impl ProtoArray {
         );
         let no_change = (parent.best_child, parent.best_descendant);
 
-        let (new_best_child, new_best_descendant) =
-            if let Some(best_child_index) = parent.best_child {
-                if best_child_index == child_index && !child_leads_to_viable_head {
-                    // If the child is already the best-child of the parent but it's not viable for
-                    // the head, remove it.
-                    change_to_none
-                } else if best_child_index == child_index {
-                    // If the child is the best-child already, set it again to ensure that the
-                    // best-descendant of the parent is updated.
-                    change_to_child
-                } else {
-                    let best_child = self
-                        .nodes
-                        .get(best_child_index)
-                        .ok_or(Error::InvalidBestDescendant(best_child_index))?;
+        let (new_best_child, new_best_descendant) = if let Some(best_child_index) =
+            parent.best_child
+        {
+            if best_child_index == child_index && !child_leads_to_viable_head {
+                // If the child is already the best-child of the parent but it's not viable for
+                // the head, remove it.
+                change_to_none
+            } else if best_child_index == child_index {
+                // If the child is the best-child already, set it again to ensure that the
+                // best-descendant of the parent is updated.
+                change_to_child
+            } else {
+                let best_child = self
+                    .nodes
+                    .get(best_child_index)
+                    .ok_or(Error::InvalidBestDescendant(best_child_index))?;
 
-                    let best_child_leads_to_viable_head =
-                        self.node_leads_to_viable_head(&best_child)?;
```

Comment:
Off this is an annoying diff due to `cargo fmt`  but the only change around here should be removing this reference

Reply:
Rustfmt may have changed its behaviour slightly too